front_end/src/actions/profileActions.js

```diff
@@ -39,6 +41,29 @@ export const createProfile = (profileData, history) => dispatch => {
     );
 };
 
+
+// Get User's Travel/Booking History
+export const getHistory = () => dispatch => {
+  // dispatch(setProfileLoading());
+  // axios
+  //   .get("/api/profile/history")
+  //   .then(res =>
+  //     dispatch({
+  //       type: GET_HISTORY,
+  //       payload: res.data
+  //     })
+  //   )
+  //   .catch(err =>
+  //     dispatch({
+  //       type: GET_HISTORY,
+  //       payload: {}
+  //     })
+  //   );
+};
+
+
+
+///// IS THIS NEEDED? I THINK WE CAN DELETE IT. IT MIGHT BE RELATED TO YUTA'S STORY FOR THE BOOKING REDUX
```

Comment:
"createDelivery" seems like a spartanstop left over

Reply:
I think it can be remove since it doesn't do anything with the response

Reply:
removed